app/modules/relationships/models.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+"""
+Relationships database models
+--------------------
+"""
+
+from sqlalchemy_utils import Timestamp
+
+from app.extensions import HoustonModel, db
+from datetime import datetime  # NOQA
+import app.extensions.logging as AuditLog
+
+import uuid
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class RelationshipIndividualMember(db.Model, HoustonModel):
+
+    guid = db.Column(
+        db.GUID, default=uuid.uuid4, primary_key=True
+    )  # pylint: disable=invalid-name
+
+    relationship_guid = db.Column(db.GUID, db.ForeignKey('relationship.guid'))
+    relationship = db.relationship(
+        'Relationship', backref=db.backref('individual_members')
+    )
+
+    individual_guid = db.Column(db.GUID, db.ForeignKey('individual.guid'))
+    individual = db.relationship('Individual', backref=db.backref('relationships'))
+
```

Comment:
wonder if you might want to throw in a **unique constraint** on `individual_guid` + `relationship_guid` so that we shouldnt get duplicates?  its likely a very rare thing, but its [pretty much a one-liner](https://github.com/WildMeOrg/houston/blob/main/app/modules/names/models.py#L68-L69).

Reply:
and if i were clever, i would know how to make a unique constraint _across the member table and relationship table_ to prevent `indivX` and `indivY` from being in **more than one** relationship of type `type`.   but i am _not_ that clever.  :)